testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestCreateDropSchema.java

```diff
@@ -54,31 +54,31 @@ public void testCreateDropSchema()
     }
 
     @Test
-    public void testDropSchemaWithLocationWithoutExternalFiles()
+    public void testDropSchemaFiles()
     {
-        String schemaName = "schema_with_empty_location_" + randomTableSuffix();
-        String schemaDir = warehouseDirectory + "/schema-with-empty-location/";
+        String schemaName = "schema_without_location_" + randomTableSuffix();
+        String schemaDir = format("%s/%s.db/", warehouseDirectory, schemaName);
 
-        onTrino().executeQuery(format("CREATE SCHEMA %s WITH (location = '%s')", schemaName, schemaDir));
+        onTrino().executeQuery(format("CREATE SCHEMA %s", schemaName));
```

Comment:
These lines aren't actually changed; I just switched the order of this test and the next after renaming them, and they're so similar that git didn't realize.